### ISSUE TITLE
Add tests of TypeScript typings

### ISSUE THREAD

**Issue by EricCrosson** [dubzzz/ava-fast-check#18]
We need some TypeScript test files for this wonderful TypeScript package :smile:

**Comment by dubzzz** [comment]
Definitely, we could maybe use the same library as the one used by fats-check itself to test them.

See: https://github.com/dubzzz/fast-check/tree/master/test/type

**Comment by EricCrosson** [comment]
Sounds like a great place to start! I haven't used `tsd` yet but it has been on my radar, I look forward to the chance to try it out
